.gitignore

```diff
@@ -8,3 +8,11 @@
 /vendor.orig/
 
 /tests/integration/rook-test/
+
+# OLM related stuff
+cluster/olm/deploy/operator.yaml
```

Comment:
These is for the generated files? Could the generation put them under a folder already ignored such as `/.work`? Or at least generating them to a single folder to ignore would be preferred.

Reply:
Actually, this is leftover from a previous iteration I believe so we can ignore `cluster/olm/deploy` entirely.